src/views/RequiredSignoffs/ViewSignoff/index.jsx

```diff
@@ -113,19 +141,25 @@ function ViewSignoff({ isNewSignoff, ...props }) {
   const handleRoleDelete = (role, index) => () => {
     const excludeRole = (entry, i) => !(i === index);
 
-    return role[2].isAdditionalRole
+    removeRole(removedRoles.concat([role]));
```

Comment:
If a user creates 2 additional roles by clicking on the "+" button twice and filling the entries then removing one of them, will `removedRoles` contain one or both roles? I was thinking maybe we could define a utility that finds the removed roles by looking at the `original` and `roles`. What do you think?

Reply:
Hmm, I'll look into this.

Are you thinking the utility would be called at the beginning of `updateRequiredSignoffs`?